### DISCUSSION TITLE
Asset metadata with single-run backfills

### DISCUSSION THREAD

**Discussion by satyamkrishna** [dagster-io/dagster#19371] [Q&A] [answered]
I am trying to use this experimental feature : [single-run backfills ](https://docs.dagster.io/concepts/partitions-schedules-sensors/backfills#launching-single-run-backfills-using-backfill-policies-). However I am not able to figure out how to give the metadata back for each partition. Currently the entire value is getting assigned to all the partitions.

**Comment by sryza** [comment, marked as answer]
Hi @satyamkrishna - this is a known limitation that's currently tracked here: https://github.com/dagster-io/dagster/issues/18123

**Reply by satyamkrishna** [reply]
Thanks for the update. I will look for the other issue
